docs/core/install/linux-ubuntu.md

```diff
@@ -35,20 +35,69 @@ The following table is a list of currently supported .NET releases and the versi
 
 [!INCLUDE [package-manager uninstall notice](./includes/linux-uninstall-preview-info.md)]
 
+## 22.10
```

Comment:
The section heading says 22.10, but the "Advisory" link goes to an issue that talks about 22.04